calypso/contracts.go

```diff
@@ -124,6 +124,51 @@ func (c ContractWrite) Spawn(rst byzcoin.ReadOnlyStateTrie, inst byzcoin.Instruc
 	return
 }
 
+// Invoke supports the following command:
+//  - update - it takes a 'data' and/or 'extraData' argument that is used to
+//    update the data and/or extradata part of the write structure.
+func (c *ContractWrite) Invoke(rst byzcoin.ReadOnlyStateTrie,
+	inst byzcoin.Instruction, cin []byzcoin.Coin) (sc []byzcoin.StateChange,
+	cout []byzcoin.Coin, err error) {
```

Comment:
Hum, that's probably being pedantic but using theses named return values make the code less readable, as one has to trace back the assignements; and in this case, it's quite easy to return `nil` on error.

Reply:
Re-wrote it without the named returns. But it looks more convoluted to me. Also, all the other `Invoke` do use named returns...

Reply:
Do you? I find it more readable as the `return` clearly indicate what is being returned and it ensure that all fields are set as intended (ie `nil` on error); do you find something specifically convoluted or only the named returns can fix it?